src/bun.js/api/bun/subprocess.zig

```diff
@@ -1434,7 +1436,7 @@ pub const Subprocess = struct {
         }
 
         if (stdin) |pipe| {
-            this.weak_file_sink_stdin_ptr = null;
+            this.island.clearStdin();
             this.flags.has_stdin_destructor_called = true;
```

Comment:
Do we still need `has_stdin_destructor_called?

Reply:
good catch